docs/data/data-grid/column-dimensions/column-dimensions.md

```diff
@@ -58,6 +58,25 @@ To capture changes in the width of a column there are two callbacks that are cal
 - `onColumnResize`: Called while a column is being resized.
 - `onColumnWidthChange`: Called after the width of a column is changed, but not during resizing.
 
+## Autosizing [<span class="plan-pro"></span>](/x/introduction/licensing/#pro-plan 'Pro plan')
```

Comment:
Let's add it to the [feature comparison](https://mui.com/x/react-data-grid/getting-started/#feature-comparison) 🙂